velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -355,9 +384,14 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
     return toVeloxPlan(readRel, outputType);
 
   } else {
-    auto tableScanNode = std::make_shared<core::TableScanNode>(
-        nextPlanNodeId(), outputType, tableHandle, assignments);
-    return tableScanNode;
+    // handle emit
+    return ProcessEmit(
```

Comment:
It doesn't seem to make sense to add emit on top of TableScan? Can we just configure TableScan to produce columns in the right order?

Reply:
Similar reason? https://github.com/facebookincubator/velox/pull/3543#discussion_r1065649491

Reply:
That's right.